repairInfoTestCase is blinking

Error:  Tests run: 12, Failures: 1, Errors: 0, Skipped: 0, Time elapsed: 36.291 s <<< FAILURE! - in com.ericsson.bss.cassandra.ecchronos.standalone.ITRepairInfo
Error:  repairInfoForRepairedInSubRanges[1]  Time elapsed: 1.898 s  <<< FAILURE!
java.lang.AssertionError: 

Expecting:
 **<0.9795918367346939>**
to be greater than or equal to:
 **<1.0>** 
	at com.ericsson.bss.cassandra.ecchronos.standalone.ITRepairInfo.repairInfoForRepairedInSubRanges(ITRepairInfo.java:227)


Error:  Failures: 
Error:    **ITRepairInfo.repairInfoForRepairedInSubRanges**:**227** 
Expecting:
 <0.9795918367346939>
to be greater than or equal to:
 <1.0> 
[INFO] 
Error:  Tests run: 38, Failures: 1, Errors: 0, Skipped: 0

--- a/standalone-integration/src/test/java/com/ericsson/bss/cassandra/ecchronos/standalone/ITRepairInfo.java
+++ b/standalone-integration/src/test/java/com/ericsson/bss/cassandra/ecchronos/standalone/ITRepairInfo.java
@@ -15,6 +15,7 @@
 
 package com.ericsson.bss.cassandra.ecchronos.standalone;
 
+import com.datastax.oss.driver.api.core.ConsistencyLevel;
 import com.datastax.oss.driver.api.core.CqlSession;
 import com.datastax.oss.driver.api.core.cql.AsyncResultSet;
 import com.datastax.oss.driver.api.core.cql.SimpleStatement;
@@ -166,13 +167,13 @@ public class ITRepairInfo extends TestBase
             stages.add(myAdminSession.executeAsync(QueryBuilder.deleteFrom("system_distributed", "repair_history")
                     .whereColumn("keyspace_name").isEqualTo(literal(tableReference.getKeyspace()))
                     .whereColumn("columnfamily_name").isEqualTo(literal(tableReference.getTable()))
-                    .build()));
+                    .build().setConsistencyLevel(ConsistencyLevel.ALL)));
             for (Node node : metadata.getNodes().values())
             {
                 stages.add(myAdminSession.executeAsync(QueryBuilder.deleteFrom("ecchronos", "repair_history")
                         .whereColumn("table_id").isEqualTo(literal(tableReference.getId()))
                         .whereColumn("node_id").isEqualTo(literal(node.getHostId()))
-                        .build()));
+                        .build().setConsistencyLevel(ConsistencyLevel.ALL)));
             }
         }
 
@@ -361,7 +362,7 @@ public class ITRepairInfo extends TestBase
                     .value("started_at", literal(Instant.ofEpochMilli(started_at)))
                     .value("finished_at", literal(Instant.ofEpochMilli(finished_at))).build();
         }
-        myAdminSession.execute(statement);
+        myAdminSession.execute(statement.setConsistencyLevel(ConsistencyLevel.ALL));
     }
 
     private Set<TokenRange> halfOfTokenRanges(TableReference tableReference)

